docgen/src/connectors.md

```diff
@@ -16,7 +16,7 @@ all the logic needed for making search widgets. Each one of them is specialized
 to make a certain type of widget.
 
 If you want to create a type of widget that is not available, you should then
-create a [custom widget]()
+create a [custom widget](https://community.algolia.com/instantsearch.js/v2/guides/customization.html).
```

Comment:
Actually this should better be a relative link so that it works in development 🤔

Reply:
Indeed.